app/models/katello/repository.rb

```diff
@@ -306,7 +306,7 @@ def self.clone_docker_repo_path(options)
                                            :content_view_id => options[:content_view]).first
         view = repo.content_view.label
         product = repo.product.label
-        env, _ = cve.label.split('/')
+        env, = cve.label.split('/')
```

Comment:
I'm not sold on this cop if we're only matching the first item. For 2+ I think it makes sense though:

```ruby
# bad
a, b, _, _ =  "this/is/a/test".split("/")

#good
a, b = "this/is/a/test".split("/")
```